NetTopologySuite/Utilities/BitTweaks.cs

```diff
@@ -48,7 +42,7 @@ internal static float ReverseByteOrder(float value)
 
         internal static double ReverseByteOrder(double value)
         {
-            return DoubleBitsConverter.Int64BitsToDouble(ReverseByteOrder(DoubleBitsConverter.DoubleToInt64Bits(value)));
+            return System.BitConverter.Int64BitsToDouble(ReverseByteOrder(System.BitConverter.DoubleToInt64Bits(value)));

```

Comment:
I think PCL didn't have the two methods that `System.BitConverter` has because Silverlight 5 didn't let you do unsafe code, which those methods rely on... now that we don't have to target PCL, we **could** outright delete `NetTopologySuite.Utilities.BitConverter`, but that would be a bit of a breaking change since the latter is public.  The `[Obsolete(..., error: true)]` approach I took should make the upgraded NTS compatible with old binaries that use our `BitConverter` for some reason, while still requiring dependents to switch to using `System.BitConverter` when building from source.